SpecFlowProjectMars1/Pages/Skill.cs

```diff
@@ -0,0 +1,201 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.DevTools.V125.DOMDebugger;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SpecFlowProjectMars1.Pages
+{
+    public class Skill
+    {
+        public void AddNewSkill(IWebDriver driver, string skillName, string skillLevel)
```

Comment:
5/
Please implement the POM Pattern, please see the below 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html